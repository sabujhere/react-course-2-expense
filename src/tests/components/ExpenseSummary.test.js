import React from 'react';
import { shallow } from 'enzyme';
import ExpenseSummary from '../../components/ExpenseSummary';

test('should connectly render expenseSummay with 1 expense', () => {    
    const wrapper = shallow(<ExpenseSummary expensesCount={1} expensesTotal={10025}/>);
    expect(wrapper).toMatchSnapshot();
  });
  

  test('should connectly render expenseSummay with 2 expenses', () => {   
    const wrapper = shallow(<ExpenseSummary expensesCount={2} expensesTotal={8000000025}/>);
    expect(wrapper).toMatchSnapshot();
  });