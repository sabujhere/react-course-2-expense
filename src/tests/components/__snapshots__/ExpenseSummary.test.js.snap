// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should connectly render expenseSummay with 1 expense 1`] = `
<div>
  Viewing 
  1
   expense
  s
   totalling 
  $100.25
</div>
`;

exports[`should connectly render expenseSummay with 2 expenses 1`] = `
<div>
  Viewing 
  2
   expense
  s
   totalling 
  $80,000,000.25
</div>
`;
